samples/x-multi-region/x-demo.tfvars

```diff
@@ -38,15 +38,18 @@ apigee_environments = {
     node_config  = null
     iam          = null
     envgroups    = ["test"]
+    type         = null
   }
   test2 = {
     display_name = "Test 2"
     description  = "Environment created by apigee/terraform-modules"
     node_config  = null
     iam          = null
     envgroups    = ["test"]
+    type         = null
   }
 }
+}
```

Comment:
I think this broke the test. Can you remove this extra '}'

Reply:
done.